app/services/pdf_service.rb

```diff
@@ -10,6 +10,7 @@ class PdfService
     emulate_media: "screen",
     launch_args: ["--font-render-hinting=medium", "--no-sandbox"],
     execute_script: "document.querySelectorAll('button').forEach(el => el.style.display = 'none')",
+    scale: 0.75 / 1.0,
```

Comment:
This doesn't fix the issue with trying to print the page on Chrome that I screenshotted before (because printing doesn't involve PdfService at all).

I've got another theory. The line breaks were occurring because when grover/puppeteer "looks at" the page, it's using a small viewport, narrow enough that the h2 line was being broken up. But it didn't actually insert visible linebreaks at those points, they only appeared in the underlying structure exposed by the accessibility tool. So we can infer that: _the size of the viewport affects accessibility data without affecting the look of the page_.

So how about you try removing the "no-wrap" CSS (to fix the print view in Chrome), but then setting the grover viewport to be super wide (the default is only 640px - https://github.com/Studiosity/grover#configuration) so that when it's viewing the page the header doesn't have a linebreak in it?

Reply:
Happy to give this a try, but the scale solution does fix the issue from your screenshot.
![Screenshot 2023-07-14 at 19 06 41](https://github.com/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/assets/16400143/4efc5c39-e697-4bf1-b947-4e361cb7fb84)

Reply:
I'm afraid it doesn't fix the issue - your screenshot above is of something different.. I'm talking about printing the results using the 'Print' button on the results page, NOT printing the PDF. As I say, this is completely unaffected by how you scale the PDF, because there is no PDF involved.

![Uploading Screenshot 2023-07-17 at 08.46.21.png…]()